docs/authentication/stytch/_category_.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
We don't need a category here. It should be one-pager, same as FusionAuth, Hanko and the others.